### ISSUE TITLE
[BUG] red-screen when using onFocus, onChange.

### ISSUE THREAD

**Issue by giautm** [erikras/redux-form#3414]
### Are you submitting a **bug report** or a **feature request**?

Bug, on react-native when using onChange, onFocus,... so on.

### What is the current behavior?

Red screen with error.

![image](https://user-images.githubusercontent.com/12751435/30432985-8307f448-998d-11e7-916b-64070c667f08.png)

### What is the expected behavior?
Work as normal

### What's your environment?

```
    "expo": "^18.0.0",
    "react": "16.0.0-alpha.12",
    "react-dom": "16.0.0-alpha.12",
    "react-native": "https://github.com/expo/react-native/archive/sdk-18.0.1.tar.gz",
    "redux-form": "^7.0.4",
```

**Comment by Aarblon** [comment]
I'm experiencing this bug as well.

**Comment by bitsal** [comment]
The same issue:
```
"expo": "^20.0.0",
"react": "16.0.0-alpha.12",
"react-native": "0.46.4",
"redux-form": "7.0.1",
```

**Comment by gustavohenke** [comment]
This was fixed in v7.1.0 by #3460, and other similar issues were open at the time: #3238, #3153
